incubator/webpagetest-server/values.yaml

```diff
@@ -0,0 +1,85 @@
+# Default values for wpt-test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: webpagetest/server
+  tag: latest
```

Comment:
Can you use a tagged version? `IfNotPresent` for a `pullPolicy` with a `latest` tag can cause problems.

Reply:
@pmeenan Can we get more tags on the webpagetest docker images? Currently there are only latest, release and fbpwa (which is 7 months old)

Reply:
@timothyclarke `latest` presents a number of challenges with `IfNotPresent`. For example, what if the test and production clusters are different. They could end up downloading and keeping two different versions. Imagine being able to debug issues there? Or, what if you want to use a newer version or even know what version you're using? The tag handling we've more recently talked about deals with this. Just want to make sure we share why it matters.